packages/console/src/components/Entities/EntityDescription.tsx

```diff
@@ -113,23 +133,32 @@ export const EntityDescription: React.FC<{
         className={styles.description}
       >
         <WaitForData
-          {...namedEntity}
+          {...descriptionEntities}
           spinnerVariant="none"
           loadingComponent={Skeleton}
         >
           <Row title={t('description')}>
-            <span
+            <div
               className={classnames({
                 [commonStyles.hintText]: !hasDescription,
               })}
             >
               {hasDescription
-                ? metadata.description
+                ? descriptionEntity?.longDescription?.value
                 : t(
                     patternKey('noDescription', entityStrings[id.resourceType]),
```

Comment:
For some reason this bit doesn't seem to be working; ran a debugger and verified `hasDescription` is `false` but it doesn't seem to render the string?

<img width="562" alt="Screenshot 2023-03-30 at 1 03 04 PM" src="https://user-images.githubusercontent.com/84735036/228951678-6fcb0490-0438-4371-a684-e85cab741eab.png">

Reply:
fixed it. 

<img width="1298" alt="image" src="https://user-images.githubusercontent.com/37936015/228982918-bd439d64-9513-4dc8-b2d1-0dd32451fcd6.png">

<img width="1280" alt="image" src="https://user-images.githubusercontent.com/37936015/228982988-bc23f4ed-ac35-436b-b0dc-11e600edae48.png">